app/controllers/auth.rb

```diff
@@ -8,7 +8,18 @@ class SlowFoodApp
   end
 
   get '/signup' do
+    erb :signup
+  end
 
+  post '/signup' do
+    @new_user = User.new(name: params['user']['username'], password: params['user']['password'], password_confirmation: params['user']['passwordconfirmation'])
```

Comment:
I would suggest you simplify this line by splitting it up like this:
```ruby
user_params = params['user']
@new_user = User.new(user_params)
```
Also, why are you using instance variables? `@new_user` is not used after the `redirect`. A local variable will do.

Reply:
Fixed